src/machines/java/com/enderio/machines/common/blockentity/solar/SolarPanelTier.java

```diff
@@ -0,0 +1,28 @@
+package com.enderio.machines.common.blockentity.solar;
+
+public enum SolarPanelTier implements ISolarPanelTier {
+
+    SIMPLE(10),
+    BASIC(40),
+    ADVANCED(80),
+    VIBRANT(160);
+
+    private final int productionRate;
+    private final int storageCapacity;
+
+    SolarPanelTier(int productionRate) {
+        this.productionRate = productionRate;
+        this.storageCapacity = productionRate * 1000;
+    }
+
+    @Override
+    public int getProductionRate() {
+        return productionRate;
+    }
+
+    @Override
+    public int getStorageCapacity() {
+        return storageCapacity;
```

Comment:
On top of using scalables, its up to you whether we just keep the 1000x scaling or if we make that configurable too.